README.md

```diff
@@ -1 +1,10 @@
-# android-mail
\ No newline at end of file
+# Infomaniak kMail
+
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
+     height="80">](https://f-droid.org/packages/com.infomaniak.mail/)
+[<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png"
+     alt="Get it on Google Play"
+     height="80">](https://play.google.com/store/apps/details?id=com.infomaniak.mail)
+
+Or get the latest APK from the [Releases Section](https://github.com/Infomaniak/android-kMail/releases/latest).
```

Comment:
We propose the apk directly on the git only for the beta
Because we would like the users to be updated automatically
This line should be removed